app/controllers/rents_controller.rb

```diff
@@ -25,6 +24,8 @@ def create
   def send_mail
     user = current_user
     title = @book[:title]
+    @from = @rent[:from]
+    @to = @rent[:to]
     RentEmailWorker.perform_async(user[:email], user[:fist_name], title, @to, user[:locale])
```

Comment:
you could simply use @rent[:to] here and avoid the variable altogether

Reply:
Done